sources/views/sideMenu.js

```diff
@@ -41,5 +41,11 @@ export default class SideMenu extends JetView {
 	}
 	init() {
 		this.use(plugins.Menu, "sideMenu:menu");
+		this.use(plugins.Locale, {
```

Comment:
locale is an app-level plugin, it should be inited in the myapp.js file on app rendering